inst/stan/epinowcast.stan

```diff
@@ -68,31 +91,63 @@ transformed data{
   // if no reporting day effects use native probability for reference day
   // effects, i.e. do not convert to logit hazard
   int ref_as_p = (model_rep > 0 || model_refp == 0) ? 0 : 1; 
+  // Type of likelihood aggregation to use
+  int ll_aggregation = likelihood_aggregation + model_miss;
```

Comment:
I wonder whether it would be better to complain to the user instead of just switching to group aggregation in case a missingness model is used... With the current approach, the user is likely unaware that a different parallelization scheme is used. Maybe we should at least warn the user (see my commit)

Reply:
Yeah good point. Part of me thinks this should just work for the user as it's a complexity they really don't need to know but I see your point of we allow it to be set then it should work as they expect with warnings.

Reply:
Really like your proposed check function. Seems like a good solution!